### PR TITLE
Twilio verify devise Readme--Migration Steps(1)

### PR DESCRIPTION
**Pull request by camckin10**
'Migration Steps' of Twilio Verify Devise (Part 1) 
Issue #83

### CODE REVIEW

**Review by wwahammy — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -1,21 +1,44 @@
...
+  -  `_authy` -> `_twilio_verify`
+  -  `authy-` -> `twilio-verify-`
+  -  `-authy` -> `-twilio-verify`
+  -  `Authy` -> `TwilioVerify`
+- Do a project file search & replace of any file with authy in the name (here's a few examples to replace)
+  - app/javascript/src/deviseTwilioVerify.js
+  - app/assets/stylesheets/devise_twilio_verify.scss
+  - config/locales/devise.twilio_verify.en.yml
+
+----------------------------------------------------------
```
[the remark this answers is not in the document]
Comment (camckin10) [reply README.md:41]: @wwahammy I am unclear about the question. Are you asking how the gem readme was created, or are you asking how the gem itself was created?

[the remark this answers is not in the document]
Comment (wwahammy) [reply README.md:41]: I'm asking how the contents of this were created.

[the remark this answers is not in the document]
Comment (camckin10) [reply README.md:41]: I believe this is from the initial readme. I can clean it up.

**Comment by camckin10** [comment]
@wwahammy  I am actually going to close this PR because the initial proper Twilio verify repo does not have this content.

**Pull request closed without merging**